vtm.js

```diff
@@ -359,6 +359,44 @@ const attributesAndSkillsMixin = {
   },
 }
 
+/**
+ * displays state of current resource assigment
+ * there are two input arrays
+ * allocatedResources array contains number of attributes assigned to that value
+ * resourceRestrictions array contains number of attributes allowed
+ * this component will display number of allocated points vs. allowed
+ * */
+app.component('restriction-state', {
+  props: ['allocatedResources', 'resourceRestrictions'],
+  computed: {    
+    lastInList() {
+      for (var i = 1; i < this.resourceRestrictions.length; i++) {  
+        if (this.allocatedResources[i] == 0 && this.resourceRestrictions[i] == 0) {
+          return i-1;
+        }
+      }
+    }    
+  },
+  template: `
+    <div class="resourceCount">
+      Divide: 
+      <span
+        v-for="(item, index) in resourceRestrictions"
+        v-show="item > 0 | allocatedResources[index] > 0"
+      >         
+        <span :class="{ red: allocatedResources[index] > item }">
+          {{ resourceRestrictions[index] - allocatedResources[index] }} x
+        </span>
+        <span v-if="index == 0" class="point">
```

Comment:
mozno v pripade  ze je to  prazdne zobrazit vsetky prazdne body(tj 5) a nie len ten prvy by mohlo byt prehladnejsie